ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -77,7 +77,7 @@ interface RegistrationTask {
   key: string;
   completionPropsKey: string;
   title: string;
-  description: string;
+  description: JSX.Element;
```

Comment:
[no action needed] I've used React.ReactNode for type-annotating in similar scenarios ([code](https://github.com/all-of-us/workbench/blob/331b1228bb7c7b8d392f2acf393c64d82e24a2ef/ui/src/app/pages/workspace/workspace-edit.tsx#L52-L52)). This seems maybe more specific / appropriate, but React.ReactNode may make it possible to avoid wrapping plain-text descriptions in <span> elements to satisfy the type-checker. Either way, we should probably aim to eventually make our type usage consistent here, and/or clarify when to use one vs. the other.

Reply:
Nice idea. I went ahead and switched to React.ReactNode for the reason you mentioned. Now the description can be a string or element.